pombola/core/fixtures/initial_data.json

```diff
@@ -0,0 +1,1952 @@
+[
```

Comment:
please rename it to demodata. initial_data sounds like it's relevant to a production environment while this is (could be) fake data.